### DISCUSSION TITLE
no decode delegate for this image format

### DISCUSSION THREAD

**Discussion by rsimon10** [ImageMagick/ImageMagick#4117] [Help]
I installed "ImageMagick-7.1.0-5-Q8-x64-dll.exe"  on my Windows 10  laptop
When I try any command, I am getting error on delegate
eg:
magick c:\test\abc.jpg c:\test\abc.png
magick: no decode delegate for this image format `JPEG' @ error/constitute.c/ReadImage/572.

Can you support in solving this issue
I tried uninstall/reinstall several times

**Comment by fmw42** [comment]
Where did you get the install? You should get the binary from https://imagemagick.org/script/download.php#windows and pick the one that matches your Windows version. Then use the installer.

**Reply by williamjacobszoon** [reply]
@rsimon10 Please also share the output of the following command

`magick -version`

**Reply by rsimon10** [reply]
@fmw42 
Yes, I got from here
https://imagemagick.org/script/download.php#windows
"ImageMagick-7.1.0-5-Q8-x64-dll.exe"


@williamjacobszoon 
>magick -version
Version: ImageMagick 7.1.0-5 Q8 x64 2021-08-22 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Visual C++: 192930133
Features: Cipher DPC Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raqm raw rsvg tiff webp xml zip zlib

**Comment by snibgo** [comment]
I have just downloaded and installed ImageMagick-7.1.0-5-Q8-x64-dll.exe on Windows 8.1.
```
Directory of f:\web\im

25/08/2021  18:40        37,900,416 ImageMagick-7.1.0-5-Q8-x64-dll.exe
```

It works fine, with no complaints when reading or writing jpg files.
```
f:\web\im>%IMG7105i%\magick -version
Version: ImageMagick 7.1.0-5 Q8 x64 2021-08-22 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Visual C++: 192930133
Features: Cipher DPC Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms
 lqr lzma openexr pangocairo png ps raqm raw rsvg tiff webp xml zip zlib
```

Perhaps there was a problem in the download process, and re-downloading and re-installing will cure it.
